### PR TITLE
feat:created product listing with filters

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

**Comment by KamiyaGaikwad** [comment]
Product listing page
image

File: ProductList/product.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.flex-columns{
+    flex-direction: column;
+}
+
+.nav-vertical{
+    grid-area: aside;
+    background-color: var(--container-border-white);
+    /* height: calc(100vh - 9%); */
```
Comment (anuj-kumary) [inline ProductList/product.css]: Remove unwanted comment

File: ProductList/product.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+#card-container{
+    background-color: transparent;
+}
+
+.wish-add{
+    background-color: var(--light-pink) !important;
+    color:var(--violet) !important;
+    border:none !important;
```
Comment (anuj-kumary) [inline ProductList/product.css]: Avoid using !important it's bad practice.

File: ProductList/productList.html [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                    <div class="radio-group">
+                    <input type="radio" class="radio" id="htl" name="radio-btn" checked />
+                    <label for="htl">Price-High to Low</label></div>
+                </div></div>
+        </nav>
+        <main class="ml mt flex-container flex-columns">
+            <h2 class="brand">Showing All products<span class="text-small">(Showing 7 products)</span></h2>
+            <div id="card-container" class="mt flex-columns">
+                <div class="card card-basic card-shadow">
+                    <img src="https://picsum.photos/id/1005/340/196" class="card-img-top" alt="...">
```
Comment (anuj-kumary) [inline ProductList/productList.html]: Give some name to **alt** attribute.

**Review by shrey27 — APPROVED** [review]

File: ProductList/product.css (hunk 1 above)
Comment (KamiyaGaikwad) [reply ProductList/product.css]: ok will do, thank you for pointing it out

File: ProductList/product.css (hunk 2 above)
Comment (KamiyaGaikwad) [reply ProductList/product.css]: Thank you for informing, i will remove it right away

File: ProductList/productList.html (hunk 3 above)
Comment (KamiyaGaikwad) [reply ProductList/productList.html]: Thank you for your suggestion, i will add name to it

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**